plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -177,6 +201,10 @@ bool chain_plugin::block_is_on_preferred_chain(const chain::block_id_type& block
    return chain().get_block_id_for_num(chain::block_header::num_from_id(block_id)) == block_id;
 }
 
+bool chain_plugin::skip_transaction_signatures() const {
```

Comment:
This function name looks like an "action", eg start skipping transaction signatures, rather than the answer to a question, "is_skipping_trx_sigs" or "should_skip_sigs".